src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { User } from '../modules/User.js';
+import { Console } from '../utils/Console.js';
+
+function validateEmail(value) {
+  const EMAIL_PATTERN = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!value) {
+    return 'Email is required';
+  }
+
+  if (!EMAIL_PATTERN.test(value)) {
+    return 'Email is not valid';
```

Comment:
Consider providing a more descriptive error message, such as 'The provided email format is invalid.'